articles/overview/what-is-quantum-computing.md

```diff
@@ -12,45 +12,51 @@ uid: microsoft.quantum.overview.what
 
 ## Quantum hardware
 
-In classical computers, bits correspond to voltage levels in silicon circuits. Quantum computing hardware can be implemented by many different physical realizations of qubits: trapped ions, superconducting, neutral atoms, electron spin, light polarization, topological qubits. Quantum hardware is an emergent technology. It requires balancing fidelity of the system with scalability. The larger the scale (that is, number of qubits), the higher the error rate.
+In classical computers, bits correspond to voltage levels in silicon circuits. Quantum computing hardware can be implemented by many different physical realizations of qubits: trapped ions, superconducting, neutral atoms, electron spin, light polarization, topological qubits. Quantum hardware is an emergent technology. Qubits are fragile by nature and become less coherent as they interact with their environment. This requires balancing fidelity of the system with scalability. The larger the scale (that is, number of qubits), the higher the error rate.
 
-Microsoft is developing a quantum computer based on topological qubits. We believe a topological qubit will be less impacted by changes in its environment, therefore reducing the degree of external error correction. Qubits are fragile by nature and become less coherent as they interact with their environment. Topological qubits feature increased stability and resistance to environmental noise, which means they can more readily scale and remain reliable longer.
+Microsoft is developing a quantum computer based on topological qubits. We believe a topological qubit will be less impacted by changes in its environment, therefore reducing the degree of external error correction. Topological qubits feature increased stability and resistance to environmental noise, which means they can more readily scale and remain reliable longer.
 
-## Quantum computing software – the Microsoft Quantum Development Kit
+## Quantum software – a full software stack 
 
-Designing a quantum program to harness interference may sound like a daunting challenge.  The Microsoft Quantum Development Kit (QDK) has been been introduced to make quantum programming and algorithm development more accessible. Our high-level programming language, Q#, addresses the challenges of quantum programming; it is integrated in a software stack that enables a quantum algorithm to be compiled down to the primitive operations of a quantum computer.
+Microsoft's quantum program is unique in that we focus on scaling each and every component of the system to deliver real quantum impact. This comprehensive approach involves: 
+* building a quantum computer using reliable, scalable and fault-tolerant topological qubits, 
+* engineering a unique cryogenic control plane with extremely low power and heat dissipation, 
```

Comment:
@bromeg  the example of the 2 balls of different colors does not at all convey the essence of entanglement (furthermore, it is used to explain what is not entanglement!). 
Entanglement is something quite unintuitive and difficult to define, but the definition I like the most is the following: 

> Two distant systems are entangled if there exist correlations between measurements on the systems that cannot be explained with "classical physics" (in more refined terminology: Local Operations and Classical Communication).

Therefore the example of the balls, that uses two classical systems, conveys precisely what entanglement is not. Entanglement is a tricky concept and for me, it was difficult to understand (that's why I was hesitant about talking about it here, although it's a very important concept it can lead to a lot of misconceptions). 
The problem in the example of the balls is that you can measure only one observable of the system (the color that you see when you open the box). But the correlations that give rise to the concept of entanglement appear when we can measure "incompatible observables" of both systems. You might be familiar with the Heisenberg uncertainty principle that states that "we cannot know the velocity and the position of a particle at the same time". This is because the position and momentum of a particle are two incompatible observables (and this incompatibility is inferred from the postulates of quantum mechanics). 
Imagine that we know the very precise position of a particle. By Heisenberg's principle, we cannot know the velocity. But it is not only that we cannot know the velocity but that the velocity of the particle is not even defined: exists in a superposition of infinite different velocities.

When we have several particles interacting, let's say two, there are also more physical laws involved, specifically conservation laws. In the case of velocity and momentum, the conservation law involved is the conservation of the momentum (velocity times mass): the total momentum of the system is conserved. Now suppose that we have a particle at rest that splits due to a, for example, nuclear decay into two particles. We know that the initial momentum of the system was 0 because the initial big particle was at rest. Momentum is conserved so both new particles must have opposite momentum. Therefore, measuring the velocity of one particle we can infer the velocity of the other, even if they're far apart! But if we measure the velocity of one particle and the position of the other we can know in principle both, position and momentum of the particle we measured the position. But wait, didn't we just say that quantum mechanics doesn't allow that? This question was brought by Einstein, Podolsky, and Rosen in an (in)famous paper and its known as the [EPR paradox](https://en.wikipedia.org/wiki/EPR_paradox).
The answer to this question is that, no, QM doesn't allow that because the particles are entangled. Entanglement means that when we measure, for example, the velocity of one of the particles and collapse its state into the measured velocity, the other particle also collapses simultaneously to have a velocity that matches the conservation law (and then the position becomes undefined)! 
Position and momentum are not the only incompatible observables, also different directions of the spin, different polarization and more are incompatible observables. Mixing measurements of different observables and doing some math you can get some correlations that cannot be explained with classical physics. The first person who did this was John Bell with its [Bell Inequalities](https://en.wikipedia.org/wiki/Bell%27s_theorem). Here you have a nice video that explains correctly what is entanglement in the context of Bell inequalities: https://www.youtube.com/watch?v=ZuvK-od647c

Don't worry if entanglement is not clear, it's a weird concept and sometimes even I doubt if I understand it myself.